### ISSUE TITLE
Move attachments into a folder:  Vault > FolderName > .attachments > NoteName > .......pdf/png/jpg etc

### ISSUE THREAD

**Issue by itoldusoandso** [dy-sh/obsidian-consistent-attachments-and-links#89]
Is there any way to move attachments into a folder:  Vault > FolderName > .attachments > NoteName > .......pdf/png/jpg etc

Currently attachments are in the same folder where the Note resides: Vault > FolderName > .......pdf/png/jpg etc

**Comment by itoldusoandso** [comment]
Nevermind, I figured it out. Need to configure properly:

_resources\${filename}

![image](https://github.com/dy-sh/obsidian-consistent-attachments-and-links/assets/71971273/b266c781-f730-4c7c-bea4-2354ea2c4a6e)

I do not need the other options so they are disabled.

Main thing is configuring the path correctly because otherwise will not work without any warning.

Now I run the command ... Collect attachments... and the attachments are moved exactly as I described above and the attachment links are updated in notes.
